Source/Scene/Cesium3DTileset.js

```diff
@@ -344,22 +354,25 @@ define([
         if (!outOfCore) {
             return;
         }
-        if (!hasAvailableRequests(tiles3D)) {
+        if (!hasAvailableRequests(tile)) {
             return;
         }
 
-        var stats = tiles3D._statistics;
-        ++stats.numberOfPendingRequests;
-        addLoadProgressEvent(tiles3D);
-
         tile.requestContent();
-        var removeFunction = removeFromProcessingQueue(tiles3D, tile);
-        when(tile.processingPromise).then(addToProcessingQueue(tiles3D, tile)).otherwise(removeFunction);
-        when(tile.readyPromise).then(removeFunction).otherwise(removeFunction);
+
+        if (!tile.isContentUnloaded()) {
+            var stats = tiles3D._statistics;
+            ++stats.numberOfPendingRequests;
+            addLoadProgressEvent(tiles3D);
```

Comment:
I would also be fine (perhaps even prefer) raising this event once per frame instead of per tile.  The former is probably better for a UI, and the later for debugging.